service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/auth/AuthenticationScopes.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.sdk.iot.service.auth;
+
+/**
+ * This class contains the most commonly used Azure Active Directory token audience scopes. They should be used as
+ * inputs when setting the authentication scopes in service client options classes such as
+ * {@link com.microsoft.azure.sdk.iot.service.RegistryManagerOptions}.
+ */
+public class AuthenticationScopes
```

Comment:
If it were C#, this class would be static. Same for java or not?

Reply:
We could also call this class IotHubAuthenticationScopes and the constants below could be simplified to DEFAULT_AUTHENTICATION_SCOPES and FAIRFAX_AUTHENTICATION_SCOPES for better readability and shorter names.

Reply:
Good idea. Since this class isn't intended to be instantiated, it does make sense to make the class static.

Reply:
Java only allows inner classes to be static. The best that I can do here is add a private no args constructor so that no user can actually construct this class.

Reply:
> We could also call this class IotHubAuthenticationScopes and the constants below could be simplified to DEFAULT_AUTHENTICATION_SCOPES and FAIRFAX_AUTHENTICATION_SCOPES for better readability and shorter names.

Sure thing